Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+authors = ["Malte Sander <malte.sander.it@gmail.com>"]
+description = "Integration tests for the Stackable Operator for Apache Kafka"
+edition = "2018"
+license = "OSL-3.0"
+name = "kafka-operator-integration-tests"
+version = "0.1.0-nightly"
```

```suggestion
version = "0.1.0"
```

I talked to @siegfriedweber  and we decided to skip the nightly tag for integration tests